src/components/molecules/SfOptions/SfOptions.stories.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { storiesOf } from "@storybook/vue";
+import { action } from "@storybook/addon-actions";
+import { linkTo } from "@storybook/addon-links";
+
+import SfOptions from "./SfOptions.vue";
+
+storiesOf("Options", module).add("with text", () => ({
+  components: { SfOptions },
+  template: '<sf-options :options="options" />',
+  data() {
+    return {
+      options: [
+        {
+          headingId: "option-label-size-1",
```

Comment:
I don't see this property being used anywhere.

Reply:
This is from mock. I will remove it soon. 😁